test.py

```diff
@@ -97,7 +97,7 @@ def target():
 parser.add_argument('--extrasimflags', default='')
 parser.add_argument('--ompython_omhome', default='')
 parser.add_argument('--noclean', action="store_true", default=False)
-parser.add_argument('--fmisimulator', default='')
+parser.add_argument('--fmisimulator', default='', help="OMSimulator is default but you can use fmpy also")
```

Comment:
Seems odd to not make it the default value, but it probably works as expected

Reply:
You mean having `default='OMSimulator'`?